test.js

```diff
@@ -205,6 +216,369 @@ test('`onEviction` option method is called after `maxSize` is exceeded', t => {
+	lru.set('1', 'test');
+	await sleep(200);
+	t.is(lru.peek('1'), 'test');
+});
+
+test('max age - peeking the item should also remove the item if it has expired', async t => {
+	const lru = new QuickLRU({maxSize: 10, maxAge: 100});
+	lru.set('1', 'test');
+	await sleep(200);
+	t.is(lru.peek('1'), undefined);
+});
+
+test('max age - peeking the item should also remove expired items that are not recent', async t => {
+	const lru = new QuickLRU({maxSize: 2, maxAge: 100});
+	lru.set('1', 'test');
+	lru.set('2', 'test');
+	lru.set('3', 'test');
+	await sleep(200);
+	t.is(lru.peek('1'), undefined);
+});
+
+test('max age - non-recent items that are not expired are also valid', async t => {
+	const lru = new QuickLRU({maxSize: 2, maxAge: 200});
+	lru.set('1', 'test');
+	lru.set('2', 'test2');
+	lru.set('3', 'test4');
+	await sleep(100);
+	t.is(lru.get('1'), 'test');
+});
+
+test('max age - has method should delete the item if expired and return false', async t => {
+	const lru = new QuickLRU({maxSize: 4, maxAge: 100});
+	lru.set('1', undefined);
+	lru.set('2', 'test');
+	lru.set('3', 'test');
+	await sleep(200);
+	t.false(lru.has('1'));
+});
+
+test('max age - has method should return the item if is not expired', t => {
+	const lru = new QuickLRU({maxSize: 4, maxAge: 100});
+	lru.set('1', undefined);
+	lru.set('2', 'test');
+	lru.set('3', 'test');
+	t.true(lru.has('1'));
+});
+
+test('max age - has method should return true for undefined value that has expiration time', t => {
+	const lru = new QuickLRU({maxSize: 2, maxAge: 100});
+	lru.set('1', undefined);
+	lru.set('2', 'test');
+	lru.set('3', 'test');
+	t.true(lru.has('1'));
+});
+
+test('max age - `.keys()` should return keys that are not expirated', async t => {
+	const lru = new QuickLRU({maxSize: 2, maxAge: 100});
+	lru.set('1', undefined);
+	lru.set('2', 'test2');
+	lru.set('3', 'test3');
+	await sleep(200);
+	lru.set('4', 'loco');
+
+	t.deepEqual([...lru.keys()].sort(), ['4']);
+});
+
+test('max age - `.keys()` should return an empty list if all items has expired', async t => {
+	const lru = new QuickLRU({maxSize: 2, maxAge: 100});
+	lru.set('1', undefined);
+	lru.set('2', 'test2');
+	lru.set('3', 'test3');
+	await sleep(200);
+
+	t.deepEqual([...lru.keys()].sort(), []);
+});
+
+test('max age - `.values()` should return an empty if all items has expired', async t => {
+	const lru = new QuickLRU({maxSize: 2, maxAge: 100});
+	lru.set('1', undefined);
+	lru.set('2', 'test2');
+	lru.set('3', 'test3');
+	await sleep(200);
+
+	t.deepEqual([...lru.values()].sort(), []);
+});
+
+test('max age - `.values()` should return the values that are not expired', async t => {
+	const lru = new QuickLRU({maxSize: 2, maxAge: 100});
+	lru.set('1', undefined);
+	lru.set('2', 'test2');
+	lru.set('3', 'test3');
+	await sleep(200);
+	lru.set('5', 'loco');
+
+	t.deepEqual([...lru.values()].sort(), ['loco']);
+});
+
+test('max age - `entriesDescending() should not return expired entries', async t => {
```

Comment:
```suggestion
test('max age - `entriesDescending()` should not return expired entries', async t => {
```

Same in many other places. "Method" calls should be wrapped in backticks.